### DISCUSSION TITLE
2021-14: 好生意与坏生意

### DISCUSSION THREAD

**Discussion by Xuanwo** [Xuanwo/blog#221] [Announcements]
https://xuanwo.io/reports/2021-14/ 

 An infrastructure engineer, focused on distributed storage system

**Comment by STRRL** [comment]
觉得这种事情还挺好玩的，好奇用户搭建起来集群跑在哪了，用来做什么了

**Comment by Xuanwo** [comment]
> 觉得这种事情还挺好玩的，好奇用户搭建起来集群跑在哪了，用来做什么了

指 SealYun 项目么？官方称有上千用户用在生产环境里面，但我个人觉得应该不会是特别严肃的场景。
